src/Services/Air/Air.js

```diff
@@ -129,5 +117,55 @@ module.exports = (settings) => {
           err => Promise.reject(new AirRuntimeError.UnableToRetrieveTickets(options, err))
         );
     },
+
+    searchBookingsByPassengerName(options) {
+      const terminal = createTerminalService(settings);
+      return terminal.executeCommand(`*-${options.searchPhrase}`)
+        .then((firstScreen) => {
+          const list = parsers.searchPaxList(firstScreen);
```

Comment:
Could you please change it to `parsers.searchPassengersList`?
We never user PAX before, so, let it be passengers.